webapp/src/shared/components/Status/Status.css

```diff
@@ -0,0 +1,36 @@
+.status {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  column-gap: 0.5rem;
+}
+
+.status-icon {
+  width: 0.625rem;
+  height: 0.625rem;
+  border-radius: 50%;
+}
+
+.status-icon-offline {
+  background: var(--color-offline);
+}
+
+.status-icon-online,
+.status-icon-playing {
+  background: var(--color-online);
+}
+
+.status-text-offline {
+  color: var(--color-offline);
+}
+
+.status-text-online {
+  color: var(--color-online);
+}
+
+.status-text-playing {
+  background: var(--gradient-game);
+  background-clip: text;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
```

Comment:
https://github.com/42AGV/ft_transcendence/blob/92a129709eb7e559fa4e1ecb253278405c917801/webapp/src/shared/components/Text/Text.css#L1-L6

This class is the same as the one from the `Text` component.
Is it a good idea to move it to index.css?

```
.status-text-playing 
.text-color-game { 	  
  background: var(--gradient-game); 	  
  background-clip: text; 	  
  -webkit-background-clip: text; 	  
  -webkit-text-fill-color: transparent; 	  
}
```

Reply:
Yes, I think so, as we have other text styles and colors, this way would be accesible to all the components, and no need to copy and paste it. It is not exaclty a color neither a style but I think we can move it there.